tests/unit/components/suite.js

```diff
@@ -0,0 +1,40 @@
+var sd = require('skin-deep'),
+    expect = require('chai').expect,
+    Suite = require('../../../ui/components/suite.jsx'),
+    fixture = require('../../fixtures/components/suite.js');
+
+describe('Suite', function() {
+  var tree;
+
+  beforeEach(function() {
+    tree = sd.shallowRender(<Suite suite={fixture}/>);
+  });
+
+  it('should indent the suite properly', function() {
+    var expected = {
+      marginLeft: Suite.MARGIN_LEFT * (fixture.indent - 1) + 'px'
+    };
+
+    expect(tree.findNode('.suite').props.style).to.be.deep.equal(expected);
```

Comment:
Compare just the `marginLeft` property. The test should be as focused as possible. If in the future you add another style property then this test will fail without a reasonable meaning.